pkg/kubectl/custom_column_printer_test.go

```diff
@@ -115,8 +122,19 @@ func TestNewColumnPrinterFromSpec(t *testing.T) {
 			t.Errorf("[%s] unexpected error: %v", test.name, err)
 			continue
 		}
+		if test.noHeaders {
+			buffer := &bytes.Buffer{}
 
-		if !reflect.DeepEqual(test.expectedColumns, printer.Columns) {
+			printer.PrintObj(&api.Pod{}, buffer)
```

Comment:
It's not gonna work because the columns are still there (which is desired behaviour), the only difference with `--no-header` happens in "presentation" method which is `PrintObj`